src/main/resources/config/liquibase/master.xml

```diff
@@ -31,6 +31,7 @@
     <include file="classpath:config/liquibase/changelog/20230419102000_changelog.xml" relativeToChangelogFile="false"/>
     <include file="classpath:config/liquibase/changelog/20230430235959_changelog.xml" relativeToChangelogFile="false"/>
     <include file="classpath:config/liquibase/changelog/20230508114600_changelog.xml" relativeToChangelogFile="false"/>
+    <include file="classpath:config/liquibase/changelog/20230519181800_changelog.xml" relativeToChangelogFile="false"/>
```

Comment:
The latest changes should be at the end, the date might sometimes deviate a bit.
Otherwise there could be inconsistencies when different instances deploy the changeling in different orders